manim/utils/opengl.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import numpy.linalg as linalg
+
+from .. import config
+
+depth = 20
+
+
+def matrix_to_shader_input(matrix):
+    return tuple(matrix.T.ravel())
+
+
+def orthographic_projection_matrix(
+    width=None, height=None, near=1, far=depth + 1, format=True
+):
+    if width is None:
+        width = config["frame_width"]
+    if height is None:
+        height = config["frame_height"]
+    projection_matrix = np.array(
+        [
+            [2 / width, 0, 0, 0],
+            [0, 2 / height, 0, 0],
+            [0, 0, -2 / (far - near), -(far + near) / (far - near)],
+            [0, 0, 0, 1],
+        ]
+    )
+    if format:
+        return matrix_to_shader_input(projection_matrix)
+    else:
+        return projection_matrix
+
+
+def perspective_projection_matrix(width=None, height=None, near=4, far=18, format=True):
+    if width is None:
+        width = config["frame_width"] / 3
+    if height is None:
+        height = config["frame_height"] / 3
+    projection_matrix = np.array(
+        [
+            [2 * near / width, 0, 0, 0],
+            [0, 2 * near / height, 0, 0],
+            [0, 0, (far + near) / (near - far), (2 * far * near) / (near - far)],
+            [0, 0, -1, 0],
+        ]
+    )
+    if format:
+        return matrix_to_shader_input(projection_matrix)
+    else:
+        return projection_matrix
+
+
+def translation_matrix(x=0, y=0, z=0):
```

Comment:
Add that wiki page you showed in stream as a comment here please.

Reply:
putting a wiki page in the code is a little weird

Reply:
I think as a comment if it helps clear things up and/or explain things then it would be good